packages/vercel/src/index.ts

```diff
@@ -468,6 +474,23 @@ export default function vercelAdapter({
 					}
 				}
 
+				let trailingSlash: boolean | undefined;
+
+				if (_config.trailingSlash && _config.trailingSlash !== 'ignore') {
+					trailingSlash = _config.trailingSlash === 'always';
```

Comment:
You might want to add a comment explaining why we assign the local variable `trailingSlash` only when `_config.tralingSlash` differs from `'ignore'`. This isn't clear from the code